csharp/ql/lib/semmlecode.csharp.dbscheme

```diff
@@ -878,6 +878,13 @@ param_location(
   int id: @parameter ref,
   int loc: @location ref);
 
+@has_scoped_annotation = @local_scope_variable
+
+scoped_annotation(
+  int id: @has_scoped_annotation ref,
+  int kind: int ref // scoped ref = 1, scoped value = 2
```

Comment:
It appears that the value of this column is not used?

Reply:
That is correct. I don't know yet whether or not we need to distinguish between scoped ref or scoped value at some later point in time, so I decided not to throw away the information.